libraries/AP_Scripting/drivers/mount-djirs2-driver.lua

```diff
@@ -0,0 +1,734 @@
+-- mount-djirs2-driver.lua: DJIRS2 mount/gimbal driver
```

Comment:
block comment is:
```
--[[
blah1
blah2
--]]
```

Reply:
txs, replaced individual line comments with block comment.